keylime/cloud_verifier_tornado.py

```diff
@@ -138,7 +138,8 @@ def verifier_read_policy_from_cache(stored_agent: VerfierMain) -> str:
         GLOBAL_POLICY_CACHE[agent_id][""] = ""
 
     if stored_agent.ima_policy:
-        checksum = stored_agent.ima_policy.checksum
+        # Old ima_policies may not have a checksum -> use "None"
```

Comment:
I will remove the comment '-> use None' since it will likely be an empty string. Per the DB upgrade script (https://github.com/keylime/keylime/blob/master/keylime/migrations/versions/2fbc0fb8fa4d_add_checksum_to_allowlist.py) checksums were added as empty columns and an empty string can be used as a key into a dict.